CHANGELOG.md

```diff
@@ -36,13 +36,17 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Updated main.less to use the paths option in less compiler.
 - Moved and renamed contact-macro to contact-layout in macros directory.
 - Moved filters macro from `post-macros.html` to `/macros/filter.html`.
-- Makes filters macro helpers private.
+- Maked filters macro helpers private.
```

Comment:
haha, oops. I maked a typo. Fixed in https://github.com/cfpb/cfgov-refresh/commit/f0befca0f0483c0ebfca3e35232cca1a2420acba